.editorconfig

```diff
@@ -0,0 +1,20 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+
+# Change these settings to your own preference
+indent_style = space
+indent_size = 2
+
+# We recommend you to keep these unchanged
+# end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+# insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
This file has been added to ensure consistent styling is used by all team members.